README.md

```diff
@@ -87,12 +34,3 @@ General instructions on _how_ to contribute should be stated with a link to [CON
 1. [TERMS](TERMS.md)
 2. [LICENSE](LICENSE)
 3. [CFPB Source Code Policy](https://github.com/cfpb/source-code-policy/)
-
-
-----
-
-## Credits and references
```

Comment:
Excellent catch! I added a reference to it at the top of the document, and succinctly referenced its influence on this document!